csharp/Platform.Collections/Segments/Segment.cs

```diff
@@ -105,9 +220,27 @@ public void CopyTo(T[] array, int arrayIndex)
             }
         }
 
+        /// <summary>
+        /// <para>Removes the first occurrence of a specific value from the <see cref="Segment"/>.</para>
+        /// <para>Удаляет первое вхождение указанного значения из <see cref="Segment"/>.</para>
+        /// </summary>
+        /// <param name="item"><para>The value to remove from the <see cref="Segment"/>.</para><para>Значение, которые нужно удалить из <see cref="Segment"/>.</para></param>
+        /// <returns></returns>
+        /// <exception cref="NotSupportedException">
+        /// <para>The <see cref="Segment"/> is read-only.</para>
+        /// <para><see cref="Segment"/> доступен только для чтения.</para>
+        /// </exception>  
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public bool Remove(T item) => throw new NotSupportedException();
 
+        /// <summary>
+        /// <para>Returns an enumerator that iterates through a <see cref="Segment"/>.</para>
+        /// <para>Возвращает перечислитель, который осуществляет итерацию по <see cref="Segment"/>.</para>
+        /// </summary>
+        /// <returns>
+        /// <para>An <see cref="T:System.Collections.IEnumerator" /> object that can be used to iterate through the collection.</para>
+        /// <para>Объект <see cref="T:System.Collections.IEnumerator" />, который можно использовать для перебора <see cref="Segment"/>.</para>
```

Comment:
`<see cref="T:System.Collections.IEnumerator" />` ⇾ `<see cref="T:System.Collections.IEnumerator"/>` (one space can be removed here and in all other places).